### PR TITLE
Added prometheus-rabbitmq-exporter helm chart

### PR DESCRIPTION
**Pull request by Juanchimienti**
**What this PR does / why we need it**:
Hi this is a new chart for a prometheus exporter for rabbitmq

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

There are no issues related 

**Special notes for your reviewer**:

This is my first chart submit

### CODE REVIEW

**Comment by Juanchimienti** [comment]
Hey people, any comments?

**Review by gianrubio — COMMENTED** [review]
/lgtm

**Comment by bradenwright** [comment]
@gianrubio @unguiculus @Juanchimienti I currently have an open PR for adding the RabbitMQ exporter as a sidecar.  I found this PR last night when a comment was made on mine https://github.com/kubernetes/charts/pull/4401

I feel that running this exporter as a sidecar so metrics/alerting/etc can autoscale with the pods is important and can't be done using a separate chart as a dependency.  For that reason I added it to the rabbitmq-ha chart.

**Comment by Juanchimienti** [comment]
@bradenwright We are running rabbit without helm right now so thats why we did it this way.

Anyhow I'm new here so I will follow what you guys thinks its best

**Comment by bradenwright** [comment]
@Juanchimienti I'm open to discussion as well, I'd rather do what works best.  But metrics on new nodes that autoscaled seemed like a need feature for running with rmq in k8s, and unless I'm wrong with this approach would need a controller or something to watch for changes and then publish a new config.  B/c of that I went with the sidecar approach.

But right now my PR would not allow deploying the exporter without rabbitmq, I could add some enable/disable options.  But not sure that would be preferred, as you have a number of configurations that are probably useful for monitoring rmq clusters outside of k8s.  This may be a situation where it makes sense to merge both.  It maybe cleaner for supporting the separate use cases (monitoring rmq running inside and outside of k8s).

**Comment by Juanchimienti** [comment]
I agree to merge both

**Comment by Juanchimienti** [comment]
@unguiculus what do you think?

File: stable/prometheus-rabbitmq-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+$ helm install incubator/prometheus-rabbitmq-exporter
+```
+
+## Introduction
+
+This chart bootstraps a [rabbitmq_exporter](https://github.com/kbudde/rabbitmq_exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
```
Comment (unguiculus) [inline stable/prometheus-rabbitmq-exporter/README.md]: 1.8+

File: stable/prometheus-rabbitmq-exporter/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+| `annotation`             | pod annotations for easier discovery                                   | {}                        |
+
+For more information please refer to the [rabbitmq_exporter](https://github.com/kbudde/rabbitmq_exporter) documentation.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
+  --set "rabbitmq.url=http://myrabbit:15672" \
+    incubator/prometheus-rabbitmq-exporter
```
Comment (unguiculus) [inline stable/prometheus-rabbitmq-exporter/README.md]: Fix references to incubator.

File: stable/prometheus-rabbitmq-exporter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        release: {{ .Release.Name }}
+      annotations:
+{{ toYaml .Values.annotations | indent 8 }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: RABBIT_URL
```
Comment (unguiculus) [inline stable/prometheus-rabbitmq-exporter/templates/deployment.yaml]: Please make list indentation consistent throughout the chart. Either indent zero or two spaces.

File: stable/prometheus-rabbitmq-exporter/values.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+loglevel: info
+rabbitmq:
+  url: http://myrabbit:15672
+  user: guest
+  password: guest
+  capabilities: bert,no_sort
+  include_queues: ".*"
+  skip_queues: "^$"
+
+annotation:
```
Comment (unguiculus) [inline stable/prometheus-rabbitmq-exporter/values.yaml]: Please comment the annotations out. They should be optional. You don't need them if you use prometheus-operator.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/prometheus-rabbitmq-exporter/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# prometheus-rabbitmq-exporter
+
+[rabbitmq_exporter](https://github.com/kbudde/rabbitmq_exporter) is a Prometheus exporter for rabbitmq metrics.
+
+## TL;DR;
+
+```bash
+$ helm install incubator/prometheus-rabbitmq-exporter
```
Comment (gianrubio) [inline stable/prometheus-rabbitmq-exporter/README.md]: s/incubato/stable

File: stable/prometheus-rabbitmq-exporter/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release incubator/prometheus-rabbitmq-exporter
```
Comment (gianrubio) [inline stable/prometheus-rabbitmq-exporter/README.md]: s/incubator/stable

File: stable/prometheus-rabbitmq-exporter/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+```bash
+$ helm install --name my-release \
+  --set "rabbitmq.url=http://myrabbit:15672" \
+    incubator/prometheus-rabbitmq-exporter
+```
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml incubator/prometheus-rabbitmq-exporter
```
Comment (gianrubio) [inline stable/prometheus-rabbitmq-exporter/README.md]: s/incubator/stable

**Comment by bradenwright** [comment]
@gianrubio @Juanchimienti this should be going in `incubator` first, doesn't seem correct to go straight to `stable` IMHO

**Comment by unguiculus** [comment]
It's ok to add it to stable.

**Comment by Juanchimienti** [comment]
This is missing the ok-to-test label. @unguiculus would you do the honors?

**Pull request merged**